plugins/modules/proxmox.py

```diff
@@ -224,7 +224,7 @@
     default: 30
   update:
     description:
-      - If V(true), the container will be updated with new values.
+      - If V(true), the container will be updated with new values. Only works with V(state: present).
```

Comment:
```suggestion
      - If V(true), the container will be updated with new values. Only works with O(state=present).
```

Reply:
Actually it also needs that `clone` has not been specified:
```suggestion
      - If V(true), the container will be updated with new values. Only works with O(state=present) and when O(clone) is not specified.
```